go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -8503,6 +8501,7 @@ func (fbo *folderBranchOps) Reset(
 		return err
 	}
 	oldHandle.SetFinalizedInfo(finalizedInfo)
+	// This can't be susceptible to the WaitGroup due to a potential deadlock.
```

Comment:
Does that mean it's still in danger of logging after shutdown?

Also, nit: I don't think the word susceptible makes sense to me here. Maybe "This call can't be included in `fbo.doneWg` due..."?

Reply:
I think I meant to type `subject` and somehow `susceptible` came out.

Reply:
The answer to your question is "yes", it is in danger of logging. Specifically here:
https://github.com/keybase/client/blob/e4e488302daad428f49db3b70395aa16ac37c63a/go/kbfs/libkbfs/kbfs_ops.go#L1444

Reply:
I'll play with it a bit more this morning to try to get rid of the deadlock.